pages/docs/reference/building-mpp-with-gradle.md

```diff
@@ -962,7 +962,7 @@ kotlin {
 </div>
 </div>
 
-Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#experimental-metadata-publishing-mode) and the project 
+Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#metadata-publishing) and the project 
 is set up to consume the metadata as well, then it is enough to specify a dependency only once, for the common source set. 
```

Comment:
`is set up to consume the metadata` – always true with Gradle 5.3+, maybe remove?

Reply:
Agree. Removed.